README.md

```diff
@@ -1,6 +1,6 @@
 # Capistrano Remote
 
-Every so often you need to look at your production data or otherwise run some manual maintenance tasks in your production Rails application.
+Ever so often you need to look at your production data or otherwise run some manual maintenance tasks in your production Rails application.
```

Comment:
"Every so often" is actually the idiom I went for here, ie in the meaning of "occasionally" or "sometimes":

* http://idioms.thefreedictionary.com/every+so+often
* http://english.stackexchange.com/questions/7003/what-is-the-correct-usage-ever-so-often-or-every-so-often